Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -666,19 +670,32 @@ for (const auto & particle_diag : particle_diags) {
         pc->getCharge(), pc->getMass(),
         isBTD, isLastBTDFlush);
     }
+}
+
+void
+WarpXOpenPMDPlot::ForceFlush(bool isBTD)
+{
+    if (!isBTD)
+        return;
 
     auto hasOption = m_OpenPMDoptions.find("FlattenSteps");
-    const bool flattenSteps = isBTD && (m_Series->backend() == "ADIOS2") && (hasOption != std::string::npos);
+    const bool result = (m_Series->backend() == "ADIOS2") && (hasOption != std::string::npos);
```

Comment:
Please keep calling this `flattenSteps`, more descriptive than `result`, especially in the `if` below
```suggestion
    const bool flattenSteps = (m_Series->backend() == "ADIOS2") && (hasOption != std::string::npos);
```

Reply:
OK

Reply:
Not yet changed :)

Reply:
Should be in there now.